master.css

```diff
@@ -36,6 +48,9 @@ td .anchor {
 td:first-child {
     min-width: 320px;
 }
+td, th {
+    border: 1px solid #fff;
```

Comment:
JSYK this also messes up the mauve outline of the "current browser" result cells.